internal/cmd/marketplace/apply/apply.go

```diff
@@ -83,32 +102,70 @@ func ApplyCmd(options *clioptions.CLIOptions) *cobra.Command {
 	return cmd
 }
 
-func applyItemsFromPaths(ctx context.Context, client *client.APIClient, restConfig *client.Config, filePaths []string) (string, error) {
+func applyItemsFromPaths(ctx context.Context, client *client.APIClient, companyID string, filePaths []string) (string, error) {
 	resourceFilesPaths, err := buildFilePathsList(filePaths)
 	if err != nil {
 		return "", err
 	}
-	applyReq, err := buildApplyRequest(resourceFilesPaths)
+	applyReq, itemNameToFilePath, err := buildApplyRequest(resourceFilesPaths)
 	if err != nil {
 		return "", err
 	}
-	outcome, err := applyMarketplaceResource(ctx, client, restConfig.CompanyID, applyReq)
+
+	for _, item := range applyReq.Resources {
+		if err := processItemImages(ctx, client, companyID, item, itemNameToFilePath); err != nil {
+			return "", err
+		}
+	}
+
+	outcome, err := applyMarketplaceResource(ctx, client, companyID, applyReq)
 	if err != nil {
 		return "", err
 	}
 
 	return buildOutcomeSummaryAsTables(outcome), nil
 }
 
-const applyEndpoint = "/api/backend/marketplace/tenants/%s/resources"
+// concatPathIfRelative concatenates the current dir of basePath only if  filePath is relative, otherwise if filePath is absolute it returns it
```

Comment:
If we need this comment, perhaps the function's name is not sufficiently expressive?